ChaosMod/Effects/db/Player/PlayerGunGame.cpp

```diff
@@ -0,0 +1,34 @@
+
+#include <stdafx.h>
+
+//Effect by ProfessorBiddle
+
+static void OnStart()
+{
+	Ped playerPed = PLAYER_PED_ID();
+	REMOVE_ALL_PED_WEAPONS(playerPed, 0);
+	GIVE_WEAPON_TO_PED(playerPed, GET_HASH_KEY("WEAPON_PISTOL"), 9999, false, true);
+}
+
+static void OnTick()
+{
+	Weapon wepGive;
+	Ped playerPed = PLAYER_PED_ID();
+	if (IS_PED_SHOOTING(playerPed))
+	{
```

Comment:
wouldnt this be TOO much chaos? GunGame usually means that you have to kill someone to get a new gun

Reply:
Maybe it'd be better if peds' guns changed too after every shot, because right now as I see it only changes for the player (for the player I think it's still better to change the gun after every shot, because with every kill it actually may become less interesting)

Reply:
I would like to do it every time the player kills someone, but I'm not exactly sure how to do that.

I had similar issues making Pacifist.